website/blog/2024-04-05-release-1.9.md

```diff
@@ -0,0 +1,221 @@
+
+Why experimental for existing users ? The main reason is that a Podman machine version 4 is not compliant at all with a Podman machine version 5.
+
+Moving to Podman v5 implies to optionally save images and then delete previous machines and create a new one.
+
+#### Important Steps before updating
+
+Prioritize data backup by using the save feature in the `Image Lists` section. This feature allows you to back up your images and restore them once you have a new Podman machine.
+
+When prompted to update, confirm to remove all existing data from your machines.
+
+For more details on the save/load feature, refer to the release notes.
+
+![v5 update](img/podman-desktop-release-1.9/podman-update-to-v5.png)
+
+**_NOTE:_** On macOS you will notice a lot of performance enhancements as Apple Hypervisor is used instead of QEMU for the Podman machine. Expect better performance on the file sharing.
+
+Here is a blog post introducing Podman v5 https://blog.podman.io/2024/03/podman-5-0-has-been-released/
+
+#### Enabling experimental flag
+
+If you're running a Podman version 4, to see the update to version 5 you need to enable a flag `v5` in Podman extension preferences.
+
+![v5 experimental flag](img/podman-desktop-release-1.9/experimental-podman-5.png)
+
+#### Onboarding notification for Podman version 5
+
+If you have previously installed Podman version 5 and Podman Desktop detects some invalid Podman machines, you'll see a notification on the dashboard to clean up old machines.
+
+![Onboarding Notification](img/podman-desktop-release-1.9/podman-require-onboarding-1.png)
+
+![Remove previous Podman machines](img/podman-desktop-release-1.9/podman-require-onboarding-2.png)
+
+### Podman 4.9.4
+
+If you want to stay on Podman v4.x, there is a new update for some bugs including a CVE/security issue.
+
+### Save/Load Images or Export/Import Containers
+
+Because Podman v5 is not able to read Podman v4 machines, before updating, backup images that you need to keep. Maybe you don't need to save images if they are all
+available on remote registries and transient.
+
+#### Save/Load Images
+
+🦭 Use Podman Desktop to save images and load them again.
+
+![Select images to save](img/podman-desktop-release-1.9/podman-save-1.png)
+
+![Save images](img/podman-desktop-release-1.9/podman-save-2.png)
+
+Load images using the Load button from the image list.
+
+![Select images to load](img/podman-desktop-release-1.9/podman-load-1.png)
+
+![Load images](img/podman-desktop-release-1.9/podman-load-2.png)
+
+#### Save/Load Containers
+
+🦭 Export filesystem of containers and import them.
+
+![Select containers to export](img/podman-desktop-release-1.9/podman-export-1.png)
+
+![Export containers](img/podman-desktop-release-1.9/podman-export-2.png)
+
+Import containers using the Load button from the image list.
+
+![Select files to import](img/podman-desktop-release-1.9/podman-import-1.png)
+
+![Import images](img/podman-desktop-release-1.9/podman-import-2.png)
```

Comment:
Import containers

Reply:
it's really an image, the container is exported as an image

Reply:
I commented b/c the png is titled Import Containers but the markdown title says Images